tests/view_models/item_view_model_test.py

```diff
@@ -0,0 +1,84 @@
+from todo_app.view_models.item_view_model import ItemViewModel
+from todo_app.models.Card import Card
+
+class TestItemViewModel:
+    @staticmethod
+    def test_getting_to_do_items_returns_only_to_do_items():
+        # Arrange
+        todo_item = Card(0, "title", "To Do")
+        doing_item = Card(0, "title", "Doing")
+        done_item = Card(1, "done", "Done")
+        model = ItemViewModel([todo_item, doing_item, done_item])
+
+        # Act
+        items = model.todo_items
+
+        # Assert
+        assert todo_item in items
+        assert doing_item not in items
+        assert done_item not in items
+
+    @staticmethod
+    def test_getting_to_do_items_when_empty_returns_empty_list():
+        # Arrange
+        model = ItemViewModel([])
+
+        # Act
+        items = model.todo_items
+
+        # Assert
+        assert len(items) == 0
+
+    @staticmethod
+    def test_getting_doing_items_returns_only_doing_items():
+        # Arrange
```

Comment:
minor, but all these `test_getting_<WHATEVER_STATUS>_items_returns_only_<WHATEVER>_items` tests are all very very similar, there's probably some way you could commonise the code. At the very least having a basic model setup somewhere